react/react1/week2/TodoList/src/components/todoApp.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import TodoList from "./TodoList";
+import TodoForm from "./TodoForm";
+
+const TodoApp = () => {
+  const [todos, setTodos] = useState([]);
+  const [time, setTime] = useState(0);
+
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      setTime((prevTime) => prevTime + 1);
+    }, 1000);
+    return () => clearInterval(intervalId);
+  }, []);
+
+  const addTodoItem = () => {
```

Comment:
It would be a good practise to follow the same naming convention that you did in the class component - ``handle<action>Todo`` with all of the handlers💡